### PR TITLE
fix translation and names

### PR DESCRIPTION
**Pull request by merlouduron**
# Pull Request Review Comments

## Overview
- **Purpose of the PR**:
    - [ ] Feature addition
    - [ ] Bug fix
    - [ ] Refactoring
    - [ ] Documentation update
    - [ ] Other (please specify below)

## Specific Areas for Verification
- Please specify any particular requests or questions for the reviewer here.

## Suggestions/Concerns
- If you have any suggestions or concerns, please mention them here.

---

_**Additional Comments for the Reviewer (Optional)**_

- Add any additional comments or information for the reviewer here.

### CODE REVIEW

File: client/src/pages/Projects/ProjectsRegistration.tsx [hunk 1]
```diff
@@ -490,7 +490,7 @@ const ProjectsRegistration = () => {
                         </div>
                         <div className='projectsRegistration_non-operating-profit-div'>
                           <label className='projectsRegistration_non-operating-profit'>
```
Comment (edward-stalley-work) [inline client/src/pages/Projects/ProjectsRegistration.tsx:492]: please change label as well!

File: client/src/pages/Projects/ProjectsRegistration.tsx [hunk 2]
```diff
@@ -411,7 +411,7 @@ const ProjectsRegistration = () => {
                         </div>
                         <div className='projectsRegistration_ordinary-profit-div'>
                           <label className='projectsRegistration_ordinary-profit'>
```
Comment (edward-stalley-work) [inline client/src/pages/Projects/ProjectsRegistration.tsx:413]: please change label as well and div classNames!

File: client/src/pages/Projects/ProjectsRegistration.tsx [hunk 3]
```diff
@@ -400,7 +400,7 @@ const ProjectsRegistration = () => {
                         </div>
                         <div className='projectsRegistration_operating-profit-div'>
                           <label className='projectsRegistration_operating-profit'>
```
Comment (edward-stalley-work) [inline client/src/pages/Projects/ProjectsRegistration.tsx:402]: please change label as well and div classNames!

File: client/src/pages/Projects/ProjectsRegistration.tsx [hunk 4]
```diff
@@ -422,7 +422,7 @@ const ProjectsRegistration = () => {
                         </div>
                         <div className='projectsRegistration_ordinary-profit-margin-div'>
                           <label className='projectsRegistration_ordinary-profit-margin'>
```
Comment (edward-stalley-work) [inline client/src/pages/Projects/ProjectsRegistration.tsx:424]: please change label as well and div classNames!

**Review by edward-stalley-work — CHANGES_REQUESTED** [review]
Thank you Merlou please see comments

File: client/src/pages/Projects/ProjectsRegistration.tsx (hunk 1 above)
Comment (merlouduron) [reply client/src/pages/Projects/ProjectsRegistration.tsx]: Fixed.

File: client/src/pages/Projects/ProjectsRegistration.tsx (hunk 3 above)
Comment (merlouduron) [reply client/src/pages/Projects/ProjectsRegistration.tsx]: Fixed.

File: client/src/pages/Projects/ProjectsRegistration.tsx (hunk 4 above)
Comment (merlouduron) [reply client/src/pages/Projects/ProjectsRegistration.tsx]: Fixed.

**Review by edward-stalley-work — APPROVED** [review]
All good thanks Merlou.

**Pull request merged**